frontend/packages/dev-console/src/components/import/advanced/AdvancedSection.tsx

```diff
@@ -31,7 +32,7 @@ const AdvancedSection: React.FC<AdvancedSectionProps> = ({ values }) => {
         <LabelSection />
       </ProgressiveListItem>
       <ProgressiveListItem name="Scale">
-        <ScalingSection />
+        {values.serverless.enabled ? <ServerlessScalingSection /> : <ScalingSection />}
```

Comment:
From the designs it says this should be under `Scaling` not `Scale`. 
@serenamarie125 can you please confirm.